jekyll/_cci2/caching.md

```diff
@@ -104,6 +104,45 @@ Here, the first key concatenates the checksum of `package-lock.json` file into t
 
 The next key does not have a dynamic component to it, it simply is a static string: `v1-npm-deps-`. If you would like to invalidate your cache manually, you can bump `v1` to `v2` in your `config.yml` file. In this case, you would now have a new cache key `v2-npm-deps`, which will trigger the storing of a new cache.
 
+### Using Caching in Monorepos
+
+There are many different approaches to utilizing caching in monorepos. The customer-provided example below shows how a user can create and then build a concatenated `package-lock` file that can be used as a checksum for the caching key. 
```

Comment:
Suggestion to add: this sort of approach can be used whenever you need to managed a shared cache based on multiple files in different parts of your monorepo

Reply:
fixed